README.md

```diff
@@ -171,6 +174,6 @@ chmod u+x /home/mes-aides-$BRANCH/deploy.sh
 
 ```sh
 ssh-add ~/.ssh/mes-aides-bot
-ssh mes-aides-$BRANCH@sgmap.fr "PORT=8200 OPENFISCA_PORT=12200 ./deploy.sh $BRANCH"
+ssh mes-aides-$BRANCH@sgmap.fr "PORT=8200 OPENFISCA_PORT=12200 ./deploy.sh"
 ssh root@sgmap.fr "service nginx reload"
```

Comment:
I ran into a case where `nginx reload` was not enough to actually take into account the new configuration. `nginx restart` was necessary.
Should we replace it here ? What makes me hesitate is:
- I'm not entirely sure this case would really happen in "real life", as it happened when I was trying things with feature branch deployment. I'm not sure either it can't happen.
- Is a `restart` more "risky" than a `reload` in term of prod ?